packages/cli-kit/src/private/node/api/graphql.ts

```diff
@@ -43,12 +42,7 @@ ${outputToken.json(error.response.errors)}
 Request ID: ${requestId}
 `
       }
-      let mappedError: Error
-      if (status < 500) {
-        mappedError = new GraphQLClientError(errorMessage, status, error.response.errors)
-      } else {
-        mappedError = new AbortError(errorMessage)
-      }
+      const mappedError: Error = new GraphQLClientError(errorMessage, status, error.response.errors)
```

Comment:
Not strictly necessary, but using the same type here allows us to retain information that is otherwise lost. This includes the specific error messages, the status code, and stack information.

Not sure of the original intent of "Client" here. Is it meant to convey a `4xx` class error, or an error received by the GraphQL client? If the former, then this new usage would require a rename, at least as currently written,.